src/client/utils/storage/ServerCookieStorage.ts

```diff
@@ -23,6 +23,8 @@ export class ServerCookieStorage implements MinimalStorage {
       signed: false,
       expires: undefined,
       path: '/',
+      sameSite: 'None',
+      secure: true,
```

Comment:
shall we make this configurable?

Reply:
You mean setting it as a .env config? Why do we need that? Setting it to secure in `localhost` works as far as I can see or am I missing something?

Reply:
Ah, or did just you mean configurable for the function? :)

Reply:
no as an env var. It just seems weird to me to set this over an insecure connection.

Reply:
But if it works locally then maybe it doesn't matter, or? I just think maybe we shouldn't use env variables unless we need to? For complexity reasons that is 🤔

Reply:
yeah this is a bit unclear to me. have read a bunch where it says it shouldn't work locally with secure cookies but it seems to do it :) I pushed a solution where we don't set it locally. but this will of course break 3d secure payments locally with the samesite flag turned on, but maybe that's ok 🙃